src/mlpack/methods/linear_regression/linear_regression.cpp

```diff
@@ -95,7 +100,9 @@ void LinearRegression::Predict(const arma::mat& points,
   {
     // We want to be sure we have the correct number of dimensions in the
     // dataset.
-    Log::Assert(points.n_rows == parameters.n_rows - 1);
+    const size_t labels = (parameters.n_rows == 0) ? size_t(0) : size_t(parameters.n_rows - 1);
```

Comment:
```suggestion
    // Prevent underflow.
    const size_t labels = (parameters.n_rows == 0) ? size_t(0) :
        size_t(parameters.n_rows - 1);
```
The line is over 80 characters. :)